src/index.js

```diff
@@ -68,6 +70,14 @@ function getUrl (config) {
   return 'http://' + config.test + '-dot-' + projectId + '.appspot.com';
 }
 
+// Delete an App Engine version
+exports.deleteVersion = function (config, done) {
```

Comment:
Decouple `deleteVersion` from the format of `config`. Also, why not use `exec`'s natural callback.

```js
exports.deleteVersion = function (version, cwd, callback) {
  exec(`gcloud app versions delete ${version} --project ${projectId} -q`, {
    cwd: cwd
  }, callback);
};
```